### PR TITLE
renamed several functions for a more consistent naming

### PR DESCRIPTION
**Pull request by romanroe**
@pburgmer @sengmann  As discussed, please take a look!

### CODE REVIEW

File: src/operators/only-instances-of.ts [hunk 1]
```diff
@@ -18,8 +18,14 @@ export function instanceOf<A>(clazz: Newable<A>) {
  *
  * @param clazz which instances are passed through
  */
-export function justInstanceOf<A>(clazz: Newable<A>) {
+export function onlyInstancesOf<A>(clazz: Newable<A>) {
```
Comment (pburgmer) [inline src/operators/only-instances-of.ts]: Should be onlyInstanceOf in my opinion. All other operators also use singular wording. It is skipNull skipNulls.

File: src/operators/skip-property-null.ts [hunk 2]
```diff
@@ -14,14 +14,20 @@ import { filter } from "rxjs/operators";
  *
  * @param property name of the property to check
  */
-export function propertyNotNull<T, P extends keyof T>(property: P) {
+export function isPropertyNotNull<T, P extends keyof T>(property: P) {
```
Comment (pburgmer) [inline src/operators/skip-property-null.ts]: @romanroe you only deprecated the operator, not the predicate function here. we expose the predicate too, and some projects them. we should provide but deprecate the name too, as you did with other predicate functions ;)

File: src/operators/skip-some-property-nil.test.ts [hunk 3]
```diff
@@ -1 +1,43 @@
-// TODO: implement tests
+import { map } from "rxjs/operators";
+import { assert } from "chai";
+import { of } from "rxjs";
+import { hasNoNilProperties, skipSomePropertyNil } from "./skip-some-property-nil";
+
+describe("hasNoNilProperties", function () {
+
+  it("works", async function () {
```
Comment (pburgmer) [inline src/operators/skip-some-property-nil.test.ts]: 'works' does not describe what the test is doing.

File: src/operators/skip-some-property-nil.test.ts [hunk 4]
```diff
@@ -1 +1,43 @@
...
+      obj.a.toExponential(); // guard is working
+      obj.b.toExponential(); // guard is working
+    }
+  });
+
+});
+
+describe("skipSomePropertyNil", async function () {
+
+  it("works", async function () {
```
Comment (pburgmer) [inline src/operators/skip-some-property-nil.test.ts]: works?

File: src/operators/skip-some-property-nil.test.ts [hunk 5]
```diff
@@ -1 +1,43 @@
...
+    const obj = {
+      a: 1 as number | null,
+      b: 2 as number | undefined,
+    };
+
+    // obj.a.toExponential(); // check: compiler error
+    // obj.b.toExponential(); // check: compiler error
+
+    if (hasNoNilProperties(obj)) {
```
Comment (pburgmer) [inline src/operators/skip-some-property-nil.test.ts]: test only tests compilation and has no expectation. but we also need a runtime test for the predicate function. ts compiler believes in our implementation.

ah, now I saw the functional test in the operator function test below. i would love to see it in the predicate test.

File: src/operators/skip-property-null.ts (hunk 2 above)
Comment (romanroe) [reply src/operators/skip-property-null.ts]: @pburgmer I dont understand. I deprecated the predicate function and didnt change the operator.

Comment (pburgmer) [reply src/operators/skip-property-null.ts]: Sorry, my fault.

**Pull request merged**